script/search/2.0/search.php

```diff
@@ -1,19 +1,22 @@
 <?php
-  /** TODO: this is wrong
-   * TODO: we need to version this endpoint and continue to return *something* on original queries
-   * https://api.keyman.com/search?q=query-string
+  /**
+   * https://api.keyman.com/search/2.0?[f=1&]q=query-string
    *
-   * Search for a keyboard. Returns a result that lists all keyboards, languages and countries that match.
+   * Search for a keyboard. Returns a result that lists all keyboards that match, based on
    * https://api.keyman.com/schemas/search.json is JSON schema
    *
-   * @param q    query-string   a partial string to search for in keyboard name, id, description, language.
-   *                            prefixes:  c:id:<id>   show languages for the country with ISO code <id>
-   *                                       l:id:<id>   show keyboards for the language with BCP 47 code <id>
-   *                                       k:id:<id>   show keyboard with the id <id>
-   *                                       c:<text>    show only countries (regions)
-   *                                       l:<text>    show only languages matching <text>
-   *                                       k:<text>    show only keyboards
-   *                                       k:legacy:<id> show keyboard with the legacy integer id <id>
+   * @param q    query-string   a partial string to search for in keyboard name, id, description, language, script, country.
+   *                            name, description, language, script, country all use full text search with decomposition of
+   *                            diacritics, normalisation etc. Ids use plain text matches.
+   *                            prefixes:  c:<name>    show keyboards for the countries matching <name>
+   *                                       c:id:<id>   show keyboards for the country with ISO 3166 <id>
+   *                                       l:<name>    show keyboards for the languages matching <name>
+   *                                       l:id:<id>   show keyboards for the language with BCP 47 code <id>.
+   *                                                   This code will be canonicalized according to langtags.json.
+   *                                       id:<id>     show keyboard with the id <id>
+   *                                       legacy:<id> show keyboard with the legacy integer id <id>
+   * @param f                   if 1, then return formatted JSON
+   * @param p                   page number to return (10 results per page) [TODO]
```

Comment:
One question:  is it possible to combine two or more prefixes in the same query, or is only one at a time supported?  I recently tried to do `id:sil_euro_latin` and `l:id:en` at the same time while experimenting with possible calls, but I couldn't figure out the right way to do it.

If only one prefix is allowed at a time, making that explicit would be nice.

Reply:
Yes, only one at a time. Remember this is a keyboard search, so `id:sil_euro_latin` is going to return a single keyboard anyway. I need to write the help.keyman.com docs for 2.0 search and will consolidate all this then.